src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsLocationMapping.java

```diff
@@ -0,0 +1,145 @@
+package uk.gov.hmcts.reform.sscs.helper.mapping;
+
+import lombok.extern.slf4j.Slf4j;
+import uk.gov.hmcts.reform.sscs.ccd.domain.CcdValue;
+import uk.gov.hmcts.reform.sscs.ccd.domain.Hearing;
+import uk.gov.hmcts.reform.sscs.ccd.domain.OverrideFields;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
+import uk.gov.hmcts.reform.sscs.exception.InvalidMappingException;
+import uk.gov.hmcts.reform.sscs.model.HearingLocation;
+import uk.gov.hmcts.reform.sscs.model.VenueDetails;
+import uk.gov.hmcts.reform.sscs.service.holder.ReferenceDataServiceHolder;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import static java.util.Objects.nonNull;
+import static org.apache.commons.lang3.ObjectUtils.isNotEmpty;
+import static uk.gov.hmcts.reform.sscs.model.hmc.reference.LocationType.COURT;
+import static uk.gov.hmcts.reform.sscs.model.hmc.reference.NextHearingVenueType.SOMEWHERE_ELSE;
+
+@Slf4j
+public final class HearingsLocationMapping {
+
+    private HearingsLocationMapping() {
+
+    }
+
+    public static List<HearingLocation> getHearingLocations(SscsCaseData caseData,
+                                                            ReferenceDataServiceHolder referenceDataServiceHolder)
+                                                            throws InvalidMappingException {
+        List<HearingLocation> locations = getOverrideLocations(caseData);
+
+        if (isNotEmpty(locations)) {
+            return locations;
+        }
```

Comment:
```suggestion
        if (isNotEmpty(overrideFields.getHearingVenueEpimsIds())) {
            return getOverrideLocations(caseData);
        }
```

Reply:
I prefer what is already there, keeps all the logic in the getOverrideLocations method